common/culture/cultures/undead.txt

```diff
@@ -536,5 +536,152 @@
 			{ name = "mercenary_company_deathfighters" }
 		}
 	}
+
+	vampiric_vrykul = {
```

Comment:
Don't forget to localize it.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Localization-Guidelines